lib/metadata/webkit.js

```diff
@@ -0,0 +1,59 @@
+"use strict";
+var bugsWebkit = require('../bugs-webkit');
+
+exports.related = function(title) {
+    return title.indexOf("WebKit export") > -1;
+};
+
+exports.verified = function(webkitIssue, wptPullRequestNumber) {
+        return bugsWebkit.get("/rest/bug/:id", { id: webkitIssue }).then(function (body) {
+            if (body && body.bugs && body.bugs.length > 0 && body.bugs[0].see_also) {
+                return body.bugs[0].see_also.some(function(url) {
+                    return url == "https://github.com/web-platform-tests/wpt/pull/" + wptPullRequestNumber;
+                });
+            }
+            return false;
+        });
+};
+
+exports.flags = function(issue) {
+        var flags = {
+            reviewed: false,
+            inCommit: false
+        };
+        return bugsWebkit.get("/rest/bug/:id/history", { id: issue }).then(function (body) {
+            body.bugs[0].history.forEach(function(history_item) {
+                // console.log("When:" + history_item.when);
+                history_item.changes.forEach(function(change) {
+                    // console.log("Change - added:" + change.added);
+                    // console.log("Change - removed:" + change.removed);
+                    // console.log("Change - field_name:" + change.field_name);
+                    if (change.field_name == "flagtypes.name") {
+                        if (change.removed.includes("review+")) {
+                            flags.reviewed = false;
+                        }
+                        if (change.removed.includes("commit-queue+")) {
+                            flags.reviewed = false;
```

Comment:
Why does taking a WebKit bug off of the commit-queue imply it is no longer reviewed?

Reply:
It is a corner case: _Commit removed from the commit queue before commit_ 


...  in other case (`cq+`  finally landed) ..., the `cq+`  flag is also removed but the issues is marked as *fixed* ( `change.added.includes("FIXED")`) so tit keeps the values in:

```
               flags.reviewed = true;
               flags.inCommit = true;
```